tests/pl/light_HassTurnOff.yaml

```diff
@@ -3,6 +3,7 @@ tests:
   - sentences:
       - "Zgaś światła w kuchni"
       - "Zgaś wszystkie światła w kuchni"
+      - "Wyłącz całe światła w kuchni"
```

Comment:
The correct form should be "Wyłącz całe światło w kuchni"

Reply:
Hmm, the whole `Całe` was added in another PR and I see the exact same issue as you do, I think this is in fact grammatically incorrect, converting `światła` to `światło` for this specific form will be somewhat problematic because it applies also for other different forms. I'll give it a go later today and try to find suitable solution, thanks for the feedback.

Reply:
hm.. I'm starting to wonder if the "całe" word fits the context of the things being run.. if we not find any correct sentence, maybe we should remove it? What do you think about it?